src/platform/silabs/ConnectivityManagerImpl_WIFI.cpp

```diff
@@ -220,6 +220,26 @@ void ConnectivityManagerImpl::_OnWiFiStationProvisionChange()
     DeviceLayer::SystemLayer().ScheduleWork(DriveStationState, NULL);
 }
 
+#if CHIP_DEVICE_CONFIG_ENABLE_SED
+CHIP_ERROR ConnectivityManagerImpl::_GetSEDIntervalsConfig(ConnectivityManager::SEDIntervalsConfig & SEDIntervalsConfig)
```

Comment:
This function should return the intervals received by the AP. `SEDIntervalsConfig.ActiveIntervalMS` and `SEDIntervalsConfig.IdleIntervalMS` will just have the same value.